docs/dataset_prepare.md

```diff
@@ -163,3 +175,29 @@ python tools/convert_datasets/stare.py /path/to/stare-images.tar /path/to/labels
 ```
 
 The script will make directory structure automatically.
+
+### A2D2
+
+To set up the A2D2 semantic segmentation dataset, first download 'Dataset - Semantic Segmentation' (`camera_lidar_semantic.tar`) from the official site ([a2d2.audi/a2d2/en/download.html](https://www.a2d2.audi/a2d2/en/download.html)). Extract the downloaded file as `a2d2/camera_lidar_semantic/`. Inside this directory, there will be 10 subdirectories, each containing separate `camera`, `label`, `lidar` subfolders.
+
+Next, while in MMSegmentation directory root, create a symbolic link from `mmsegmentation/data/a2d2` to the dataset directory `a2d2/camera_lidar_semantic`.
+
+```shell
+ln -s data/a2d2/ /absolute/path/to/a2d2/camera_lidar_semantic/
+```
+
+Finally, convert the A2D2 dataset to the MMSegmentation format using either segmentation category labels. Note that the dataset path should be the absolute path, NOT the previously generated symbolic link.
+
+```shell
+# For A2D2 semantic category labels
+python tools/convert_datasets/a2d2.py /absolute/path/to/a2d2/camera_lidar_semantic
+```
+
+The default arguments will result in randomly splitted 'train' and 'val' sets, each consisting of 98% and 2% of all samples, respectively. Additionally, add `--nproc N` for multiprocessing using N threads.
+
```

Comment:
2% is too little for validation.
Maybe we can use 15~20% for validation.

Reply:
Changed back to randomly sampled train/val/test splits. Default ratios are according to the A2D2 paper experiment description (70.0%, 10.3%, 19.7%).

The reason I originally settled for 2% is that the large amount of evaluation samples made me run out of memory. Please let me know what you think about the current 4188 (val) + 7897 (test) evaluation sample split. I have not tried running an evaluation yet.

Reply:
> The reason I originally settled for 2% is that the large amount of evaluation samples made me run out of memory.
Try `efficient_test`  #699 

> Changed back to randomly sampled train/val/test splits. Default ratios are according to the A2D2 paper experiment description (70.0%, 10.3%, 19.7%).
That's reasonable. 
But we may not use a random split, since the train-val-test set will not be the same for each data pre-process.
We can set the random seed or give train-val-test set list.

Reply:
> Try efficient_test #699

OK, will check out later.

> We can set the random seed or give train-val-test set list.

Have `random.seed(14)` after the imports, so the random split should be consistent each run, right.